pkgs/development/python-modules/conda/default.nix

```diff
@@ -15,6 +16,8 @@ buildPythonPackage rec {
   pname = "conda";
   version = "4.3.16";
 
+  disabled = pythonAtLeast "3.10";
```

Comment:
the version packaged in nixpkgs is extremely outdated (6 years old) and is incompatible with python 3.10+. It is probably already fixed upstream, but I didn't want to try to update it, so I just decided to disable it for these python versions.

Reply:
Can we add a comment that's because we are outdated and we terribly need to bump our package (so no people gets confused and have to discover the 6 years old thing after some thinking)?